src/api/binaries.rs

```diff
@@ -50,6 +50,7 @@ pub struct Binary {
     pub organization_prn: String,
     pub prn: String,
     pub inserted_at: String,
+    pub revision: u64,
```

Comment:
Could revision go so high to be u64? all in all, it does not matter that much the type for the CLI but still feels a bit weird to be a u64

Reply:
Agree. Does u32 feels more appropriate?